doccano_client/beta/controllers/relation.py

```diff
@@ -66,3 +66,26 @@ def all(self) -> Iterable[RelationController]:
                 relations_url=self.relations_url,
                 client_session=self.client_session,
             )
+
+    def create(self, relation: Relation) -> RelationController:
+        """Create a new relation, return the generated controller
+
+        Args:
+            relation: Relation. See mock_api_responses for the desired format.
```

Comment:
If it's not too much to ask might good to add the format here quickly, saving future contributors/readers of code from having to look at another file.